pkg/apis/performanceprofile/v1/doc.go

```diff
@@ -0,0 +1,4 @@
+// +k8s:deepcopy-gen=package
+
+// Package v1 is the v2 version of the API.
+package v1
```

Comment:
Package v1 is the v2 version of the API -> Package v1 is the `v1` version of the API  ?

Reply:
thanks, done